acceptance/tests/ticket_1123_facter_with_invalid_locale.rb

```diff
@@ -0,0 +1,7 @@
+test_name "ticket 1123  facter should not crash with invalid locale setting"
+confine :except, :platform => 'windows'
+agents.each do |host|
+  step "set an invalid value for locale and run facter"
+  on host,( "LANG=ABCD facter facterversion"), :acceptable_exit_codes => [ 0 , 2 ]
```

Comment:
Hmm...didn't know facter exit code was different from puppet exit code. 
I will also add a check for "WARN" in the stdout and null in stderr